cli/data.py

```diff
@@ -281,24 +281,39 @@ def update_case_based_icu_utilization_weights():
 
 
 @main.command()
-def update_test_combined_data():
+@click.option(
+    "--truncate-dates/--no-truncate-dates",
```

Comment:
What's the motivation behind adding this functionality? Will there be multiple test outputs with different data sets of test data? 

Would be good to indicate somewhere why/when this should be used